security_center/incident_handling_process.rst

```diff
@@ -1,3 +1,5 @@
+.. _psirt_process:
+
```

Comment:
Do you need to create a new reference? Isn't a doc reference enough? e.g. :doc:`incident_handling_process`